extensions/amp-base-carousel/0.1/test-e2e/test-arrows-non-looping.js

```diff
@@ -45,27 +42,30 @@ describes.endtoend('AMP carousel arrows when non-looping', {
   beforeEach(async() => {
     controller = env.controller;
 
-    prevArrowSlot = await getPrevArrowSlot(controller);
-    nextArrowSlot = await getNextArrowSlot(controller);
+    prevArrow = await getPrevArrow(controller);
     nextArrow = await getNextArrow(controller);
   });
 
   it('should have the arrows in the correct initial state', async() => {
-    await expect(css(prevArrowSlot, 'opacity')).to.equal('0');
-    await expect(css(nextArrowSlot, 'opacity')).to.equal('1');
+    await expect(css(prevArrow, 'opacity')).to.equal('0');
+    await expect(css(nextArrow, 'opacity')).to.equal('1');
   });
 
   it('should show the prev arrow when going to the first slide', async() => {
     await controller.click(nextArrow);
-    await expect(css(prevArrowSlot, 'opacity')).to.equal('1');
-    await expect(css(nextArrowSlot, 'opacity')).to.equal('1');
+    await expect(css(prevArrow, 'opacity')).to.equal('1');
+    await expect(css(nextArrow, 'opacity')).to.equal('1');
   });
 
   it('should hide the next arrow when going to the end', async() => {
-    const el = await getScrollingElement(controller);
-    await controller.scrollBy(el, {left: SLIDE_COUNT * pageWidth});
+    // click to the end instead of using controller.scrollBy()
+    let clicks = 0;
+    while (clicks < SLIDE_COUNT) {
+      await controller.click(nextArrow);
```

Comment:
Should we reduce the number of slides so that we don't have to wait for the animation seven times?

Reply:
Done!